app/src/organisms/InstrumentMountItem/__tests__/ProtocolInstrumentMountItem.test.tsx

```diff
@@ -33,6 +63,7 @@ describe('ProtocolInstrumentMountItem', () => {
       attachedCalibrationData: null,
       speccedName: 'p1000_multi_gen3',
     }
+    mockPipetteWizardFlows.mockReturnValue('pipette wizard flow' as any)
```

Comment:
how come you type the return value as any instead of doing this?
```suggestion
    mockPipetteWizardFlows.mockReturnValue(<div>pipette wizard flow<div>)
```

Reply:
no reason really! it has the same effect in the test. I'll update to an element though